content/TruenasApps/StableApps/InstallNextCloudMedia.md

```diff
@@ -15,151 +15,276 @@ keywords:
+
+* Set up a Nextcloud account.
+  If you have an existing Nextcloud account, you enter the credentials for that users in the installation wizard.
+  If do not have an existing Nextcloud account you can create the account from the application install wizard.
+
+### Installing the Nextcoud App
+{{< hint info >}}
+This basic procedure covers the required Nextcloud app settings.
+For optional settings, see [Understanding App Installation Wizard Settings](#understanding-app-installation-wizard-settings).
+{{< /hint >}}
 
-### Installing Nextcloud in TrueNAS
-Go to **Apps**.
-If the pool for apps is not already set, do it when prompted.
+{{< include file="/static/includes/apps/AddMultipleAppInstancesAndNaming.md" >}}
 
-{{< trueimage src="/images/SCALE/Apps/AppsChooseAPoolForApps.png" alt="Choose A Pool for Apps" id="Choose A Pool for Apps" >}}
+{{< include file="/static/includes/apps/LocateAndOpenInstallWizard.md" >}}
 
-   When set, the **Installed Applications** screen displays **Apps Service Running** on the top screen banner.
+{{< trueimage src="/images/SCALE/Apps/InstallNextcloudScreen.png" alt="Install Nextcloud Screen" id="Install Nextcloud Screen" >}}
 
-1. Click **Discover Apps** and then locate the **Nextcloud** app.
-   Change the **Sort** to **App Name**, then type **Nextcloud** in the search field to display the app widget.
+{{< include file="/static/includes/apps/AppsWizardAppNameAndVersion.md" >}}
 
-   {{< trueimage src="/images/SCALE/Apps/LocateNextcloudAppWidget.png" alt="Locate Nextcloud App Widget" id="Locate Nextcloud App Widget" >}}
+Next, enter the **Nextcloud Configuration** settings.
+{{< expand "Entering Nextcloud Configuration Settings" "v" >}}
+For a basic installation, you can accept default values, but enter the following values:
+Enter the name of the administration user in **Admin User** and **Admin Password**.
+If using an existing Nextcloud account, enter the administration credentials for that account or enter new to create new Nextcloud user account credentials.
 
-   Click on the widget to open the **Nextcloud** details screen, then click **Install**.
-   If this is the first application installed, TrueNAS displays a dialog about configuring apps.
+{{< trueimage src="/images/SCALE/Apps/InstallNextcloudConfig1.png" alt="Nextcloud Configuration Username" id="Nextcloud Configuration Username" >}}
 
-   {{< trueimage src="/images/SCALE/Apps/AppsInformationDialog.png" alt="Apps Information Dialog" id="Apps Information Dialog" >}}
+Select the APT packages you want to use. Nextcloud requires ffmpeg and smbclient. Click **Add** to the right of **APT Packages** twice to add two sets of **Package** fields.
+Select **ffmpeg** in one, and **smbclient** in the other.
+If also selecting **ocrmypdf**, also select the **Tesseract Language Codes** option you want to use. Click **Add** to show the **Language** field then select either **chi-sim** for simplified Chinese or **eng** for English.
```

Comment:
On my system **Language** is just a text entry field, no dropdown with language options.

https://ocrmypdf.readthedocs.io/en/latest/languages.html#installing-additional-language-packs
Says that typically English is supported by default but you may have to install other language packs? Might want to check in with Stavros on how this field works and then link to either that page or https://github.com/tesseract-ocr/tesseract/blob/main/doc/tesseract.1.asc#languages for more on language choices.